docs/angular/fundamentals/monorepos-automation.md

```diff
@@ -341,6 +342,18 @@ The `tags` array is used to impose constraints on the dependency graph. Read mor
 
 Nx uses its advanced code analysis to construct a dependency graph of all applications and libraries. Some dependencies, however, cannot be determined statically. You can use the `implicitDependencies` array to list the dependencies that cannot be determined statically.
 
+The `ignore` property is _optional_, but can defined to ignore additional files and folders during the `affected:*` commands.
```

Comment:
I'd remove this from this doc. It's a fundamentals doc that is supposed to give an overview of the whole monorepo story. I think the `ignore` is a bit more advanced. And just seeing the ignore property gives folks enough of an idea that the feature is there.